src/blockdata/transaction.rs

```diff
@@ -426,6 +426,13 @@ impl Transaction {
         self.get_scaled_size(1)
     }
 
+    /// Gets the "vsize" of this transaction. Will be `ceil(weight / 4.0)`.
+    #[inline]
+    pub fn get_vsize(&self) -> usize {
+        let weight = self.get_weight();
+        (weight / WITNESS_SCALE_FACTOR) + if weight % WITNESS_SCALE_FACTOR == 0 { 0 } else { 1 }
```

Comment:
The usual (and more efficient) way to divide-rounding-up is to add one minus the divisor and then divide, instead of branching.

Reply:
Thanks for a comment. I fixed this.